README.md

```diff
@@ -25,3 +25,9 @@ To simplify dependency management, rbt vendors a copy of Roc's standard library
 We keep this in sync with the Roc version from niv, but check in the files to make life easier in CI.
 
 To update this code, get into a dev shell and run `sync-roc-std`, then commit the changes.
+
+### Updating `src/glue.rs`
+
+Regenerate bindings between Roc (whose entrypoint is `Package-Config.roc`) and `src/glue.rs` by running `sync-glue` from a dev shell.
```

Comment:
🎨  For what it's worth, nowadays this is usually named something like `platform/main.roc` instead of `Package-Config.roc` - but either works!

Reply:
mm, good to know. I'll make a PR to change that. I think it's probably worth being normal here as much as possible since this is likely gonna be an early large platform that folks might look at to figure out how things are done.